src/main/resources/META-INF/accesstransformer.cfg

```diff
@@ -0,0 +1 @@
+public net.minecraft.client.renderer.entity.EntityRendererManager field_178637_m
```

Comment:
For readability purposes, I'd suggest adding the mapped name for this.

Reply:
No longer using the AT

Reply:
AT can be removed, as per above instruction.

Reply:
Yeah, I made the review comment before the AT was removed, and GitHub _helpfully_ didn't show the comment to me again for me to remove.